.github/workflows/benchmark.yml

```diff
@@ -110,15 +110,17 @@ jobs:
     - name: Get only results
       run: |
         grep '^Detected Credentials: \|^result_cnt : \|^credsweeper ' credsweeper.log | LC_ALL=C sort | tee benchmark.txt
-        cp $(find . -type f -name "output.json") .
+        cp -vf $(find . -type f -name "output.json" -print -quit) report.json
 
     - name: Upload artifact
+      if: always()
       uses: actions/upload-artifact@v3
       with:
         name: report
-        path: output.json
+        path: report.json
 
     - name: Upload artifact
+      if: always()
```

Comment:
Why this field is needed?
Doesn't it run every time without this field?

Reply:
``cp $(find . -type f -name "output.json") .`` generated error when output file is the same.
Hope, it will not happen but the steps perform to upload artifacts if they exists.
They might be valuable.

Reply:
Yeah I see. But what relation between `always()` and checking if they exists?

Reply:
it is skipped when 'cp' fails.

Reply:
Thank you for explanation.